pandas/tests/io/test_packers.py

```diff
@@ -843,13 +844,13 @@ def test_default_encoding(self):
             assert_frame_equal(result, frame)
 
 
-def legacy_packers_versions():
```

Comment:
Two main changes:

This used to be called in a `pytest.mark.parametrize`. Now we build the list ahead of time, and use a parametrized fixture.

`legacy_packers_versions` used to return a *directory*. Now the parametrized fixture returns each file.

Reply:
Here's a local run:

```
pytest pandas/tests/io/test_pickle.py::test_pickles -rsx                   
==================================================================== test session starts ====================================================================
platform darwin -- Python 3.6.4, pytest-3.4.1, py-1.5.2, pluggy-0.6.0
rootdir: /Users/taugspurger/sandbox/pandas-ip/pandas, inifile: setup.cfg
plugins: xdist-1.20.1, repeat-0.4.1, forked-0.2, cov-2.5.1, hypothesis-3.44.6
collected 53 items

pandas/tests/io/test_pickle.py .....................................................                                                                  [100%]

================================================================= 53 passed in 2.68 seconds =================================================================
```


err sorry that was for pickle, but they had the same refactoring.